Cargo.toml

```diff
@@ -54,7 +54,7 @@ cfg-if = { version = "0.1.10" } # if cfg(..)  { ...  }
 chrono = { version = "0.4.11", optional = true } # compare dates etc
 
 # https://github.com/soc/dirs-rs
-dirs = { version = "3.0.1", optional = true } # get cache dirs to look for sccache cache
+dirs-next = { version = "1.0.1", optional = true } # get cache dirs to look for sccache cache
```

Comment:
The link to the github repo of the project needs to be updated as well.

Reply:
Will do. Thanks for the catch, I totally missed that.